mutt_menu.h

```diff
@@ -118,7 +118,7 @@ struct Menu
    * custom_redraw - Redraw the menu
    * @param menu Menu to redraw
    */
-  void (*custom_redraw)(struct Menu *menu);
+  void (*custom_redraw)(struct Menu *menu, struct Mailbox *m, int msg_in_pager);
```

Comment:
Hmm...  This is tricky.
We don't want the `Menu` to know any more than it needs to, so I'm very suspicious of `msg_in_pager`.
Can we put this info in `Menu->redraw_data`, making it private to the Pager?

Another problem is that both Index and Compose use `custom_redraw()`, but you haven't altered their callback functions!